src/maintainer/conda_forge_yml.rst

```diff
@@ -302,7 +302,8 @@ conda_solver
 ------------
 
 Choose which ``conda`` solver plugin to use for feedstock builds.
-The default is ``libmamba``. Other values include ``classic``.
+Note this configuration might :ref:`conda_build_tool` (e.g. when set to ``conda-build``)
```

Comment:
I feel like this is missing a verb after `might`

Reply:
Oops, true. Thanks!